Gitlab/two-stage-pipeline-template/{{cookiecutter.outputDir}}/.gitlab-ci.yml

```diff
@@ -0,0 +1,149 @@
+variables:
+  SAM_TEMPLATE: {{cookiecutter.sam_template}}
+  AWS_ACCESS_KEY_ID_: ${AWS_ACCESS_KEY_ID}
+  AWS_SECRET_ACCESS_KEY_: ${AWS_SECRET_ACCESS_KEY}
+  TESTING_STACK_NAME: {{cookiecutter.testing_stack_name}}
+  TESTING_REGION: {{cookiecutter.testing_region}}
+  TESTING_PIPELINE_EXECUTION_ROLE: {{cookiecutter.testing_pipeline_execution_role}}
+  TESTING_CLOUDFORMATION_EXECUTION_ROLE: {{cookiecutter.testing_cloudformation_execution_role}}
+  TESTING_ARTIFACTS_BUCKET: {{cookiecutter.testing_artifacts_bucket}}
+  TESTING_ECR_REPO: {{cookiecutter.testing_ecr_repo}}
+  PROD_STACK_NAME: {{cookiecutter.prod_stack_name}}
+  PROD_REGION: {{cookiecutter.prod_region}}
+  PROD_PIPELINE_EXECUTION_ROLE: {{cookiecutter.prod_pipeline_execution_role}}
+  PROD_CLOUDFORMATION_EXECUTION_ROLE: {{cookiecutter.prod_cloudformation_execution_role}}
+  PROD_ARTIFACTS_BUCKET: {{cookiecutter.prod_artifacts_bucket}}
+  PROD_ECR_REPO: {{cookiecutter.prod_ecr_repo}}
+  DOCKER_DRIVER: overlay2
+  # Create the certificates inside this directory for both the server
+  # and client. The certificates used by the client will be created in
+  # /certs/client so we only need to share this directory with the
+  # volume mount in `config.toml`.
+  DOCKER_TLS_CERTDIR: "/certs"
+
+image: docker:19.03.15
+
+services:
+    - docker:19.03.15-dind
+
+before_script:
+  - apk update
+  - apk upgrade
+  - apk add --update --no-cache python3
+  - apk add --update py-pip
+  - apk add python3-dev
+  - apk add build-base
+  - pip install awscli
+  - pip install aws-sam-cli
+
+stages:
+  - unit-test
+  - build
+  - testing
+  - prod
+
+# uncomment and modify the following step for running the unit-tests
+#
+#unit-test:
+#  stage: test
+#  only:
+#    - {{cookiecutter.main_git_branch}}
+#    - /^feature-.*$/
+#  script: |
+#    - Assuming python runtime
+#    - pip install pytest
+#    - pip install -r /path/to/requirements.txt
+#    - python -m pytest /path/to/unit-tests
```

Comment:
let's remove this part to be aligned with the other two templates

Reply:
https://github.com/aws/aws-sam-cli-pipeline-init-templates/blob/main/Jenkins/two-stage-pipeline-template/%7B%7Bcookiecutter.outputDir%7D%7D/Jenkinsfile#L35-L43

Reply:
Yep will remove this part.